### DISCUSSION TITLE
Visual Studio, .net8, cleans app data every build.

### DISCUSSION THREAD

**Discussion by SergTomcat** [dotnet/maui#18840] [Q&A]
After upgrading project to .net8, every build cleans all app's local data from previous session on device, app lauches fresh new each time.

Is this a new feature and how to disable it?

**Comment by JRosanowski** [comment]
I haven't tried .net8 so it may be some type of bug but keeping the app data is usually controlled by an option in the Visual Studio settings. "Preserve application data cache on device between deploys"

![image](https://github.com/dotnet/maui/assets/14955434/aec1b1d3-b215-4650-834e-f493d0ed4bd8)

**Comment by SergTomcat** [comment]
@JRosanowski tried it, even turned it on and off, still doesn't preserve.
It happened just after VS upgrade to 17.8.0

**Comment by xXComanderXx** [comment]
I have the same issue.

**Reply by latala43** [reply]
I have the same issue.

**Comment by Yannikk1996** [comment]
Can also reproduce this issue

**Comment by TimothyFran** [comment]
Having same problem
